packages/node/src/login/oidc/oidcHandlers/RefreshTokenOidcHandler.ts

```diff
@@ -93,10 +93,14 @@ async function refreshAccess(
       ...refreshOptions,
       refreshToken: tokens.refreshToken ?? refreshOptions.refreshToken,
     };
+    const expiresInS =
+      tokens.expiresIn ??
+      (tokens.expiresAt ?? 0) - Math.trunc(Date.now() / 1000);
     const authFetch = await buildAuthenticatedFetch(tokens.accessToken, {
       dpopKey,
       refreshOptions: keepAlive ? rotatedRefreshOptions : undefined,
       eventEmitter,
+      expiresIn: expiresInS > 0 ? expiresInS : undefined,
```

Comment:
This missing was a bug resulting in the session expiration date being incorrect.